resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -ex
```

Comment:
I know -x can produce a lot of output, but we lose being able to see what controller.sh is actually doing when it executes.